### DISCUSSION TITLE
Unexpected error: Cannot find preset's package (@)

### DISCUSSION THREAD

**Discussion by JounQin** [renovatebot/renovate#8422] [Request Help]
**What Renovate type, platform and version are you using?**

GitHub bot

**Describe the bug**

I have https://unpkg.com/browse/@1stg/renovate-config@0.7.0/package.json with `renovate-config` configed, but the bot still complains.

**Relevant debug logs**

https://github.com/rx-ts/babel-preset-proposal-typescript/issues/114

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

https://github.com/rx-ts/babel-preset-proposal-typescript/issues/114

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [x] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

**Comment by rarkins** [comment]
Please change `"extends": "@1stg"` to `"extends": ["@1stg"]`

**Reply by JounQin** [reply]
Thanks for your reply, `"extends": "@1stg"` worked before, why the behavior was changed?

**Reply by viceice** [reply]
@rarkins Shouldn't the single string work too? We ave `allowString=true` 🤔 

https://github.com/renovatebot/renovate/blob/d15933d38255d15bb0bdab6f7b53196beb72936b/lib/config/definitions.ts#L193-L202

**Reply by rarkins** [reply]
Let's take a look if we get a minimal reproduction

**Reply by JounQin** [reply]
How should I provide the minimal reproduction? https://github.com/rx-ts/babel-preset-proposal-typescript/issues/114 is always been reopened by the bot.

**Reply by rarkins** [reply]
Please read the link about minimal reproductions at the tip of this thread
